third_party/move/move-prover/tests/sources/functional/resource_index.exp

```diff
@@ -0,0 +1,12 @@
+Move prover returns: exiting with model building errors
+error: undeclared `test::Y`
```

Comment:
Is this expected?  I could use some comments on the tests to tell me what we think should happen.